### DISCUSSION TITLE
中山大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#8] [University]
> 回答来自群友，我仅负责搬运

## 东校

**宿舍是上床下桌吗**
可能四改六，即原上床下桌有两个桌改成床，过道摆桌子

**教室和宿舍有没有空调**
有

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
有，放假后需要去有隔间的澡堂洗澡，在生活区里

**有早自习、晚自习吗**
无

**有晨跑吗**
无

**每学期跑步打卡的要求是多少公里，可以骑车吗**
无

**寒暑假放多久，每年小学期有多长**
寒假3.5周，暑假不足两个月，两学期制

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，疫情前在宿舍楼下

**学校附近有地铁站吗**
十分钟走路

**宿舍楼有洗衣机吗**
自己买

**校园网怎么样，每天断电断网吗，几点开始断**
校园网包办，不断冷水电空调网

**食堂价格贵吗，会吃出异物吗**
分档次，不怎么会

**洗澡热水供应时间**
晚上凌晨之前

**校园内可以骑电瓶车吗，电池在哪能充电**
似乎是可以的

**Comment by CrossingX** [comment]
喜欢中大的自由, 比较靠自觉
